src/main.rs

```diff
@@ -54,7 +55,12 @@ impl App {
 
 fn run(run_interactive: bool) -> bool {
     println!("{} Starting!", PREFIX);
-    let config: Config = toml::from_str(&fs::read_to_string("config.toml").unwrap()).unwrap();
+
+    let mut config_path = dirs::config_dir().unwrap();
+    config_path.push("topclean.toml");
```

Comment:
I think this line needs to push `topclean` if the resulting path is supposed to be `$XDG_HOME/.config/topclean/config.toml`.